Add introduction for animation section (#5207)
godotengine/godot-docs: tutorials/animation/index.rst

Animation
=========

> This section of the tutorial covers using the two animation nodes in godot
> and the animation editor.

Reviewer: ```suggestion
This section of the tutorial covers using the two animation nodes in Godot
and the animation editor.
```

   cutout_animation
   2d_skeletons
   animation_tree

> For information on how to import animations with a 3D model see :ref:`doc_importing_3d_scenes`

Reviewer: ```suggestion
See :ref:`doc_importing_3d_scenes` for information on importing animations from a 3D model.
```
